tests/ImageSharp.Tests/Processing/Transforms/TransformTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SixLabors.ImageSharp.Tests.Processing.Transforms
+{
+    using System.Numerics;
+    using System.Reflection;
+
+    using SixLabors.ImageSharp.PixelFormats;
+    using SixLabors.ImageSharp.Processing;
+    using SixLabors.Primitives;
+
+    using Xunit;
+
+    public class TransformTests : FileTestBase
+    {
+        public static readonly TheoryData<float, float, float> TransformValues
+            = new TheoryData<float, float, float>
+                  {
+                      { 20, 10, 45 },
+                      { -20, -10, 45 }
+                  };
+
+        public static readonly List<string> ResamplerNames
+            = new List<string>
+                  {
+                      nameof(KnownResamplers.Bicubic),
+                      //nameof(KnownResamplers.Box),
```

Comment:
Do we need those commented out lines?

Reply:
Depends on how we do the testing. We really should have an expected output + comparison to prevent regression bugs for all these but I wanted approval of the output before I created standard files.

Reply:
We probably should keep these here, in the generic transform test class + drop them in `RotateTest` and `SkewTest`.